baby-gru/src/utils/MoorhenMoleculeRepresentation.ts

```diff
@@ -617,17 +617,13 @@ export class MoorhenMoleculeRepresentation implements moorhen.MoleculeRepresenta
         const atomColours = {}
         selectedGemmiAtoms.forEach(atom => { atomColours[`${atom.serial}`] = colour })
         let sphere_size = 0.3
-        let click_tol = 0.65
-        if (this.parentMolecule.representations.some(item => item.style === 'VdwSpheres' && item.visible)) {
-            sphere_size = 1.8
-            click_tol = 3.7
-        }
         let objects = [
             gemmiAtomsToCirclesSpheresInfo(selectedGemmiAtoms, sphere_size, "PERFECT_SPHERES", atomColours)
         ]
         objects.forEach(object => {
-            object["clickTol"] = click_tol
+            object["clickTol"] = 1e-6
             object["doStencil"] = true
+            object["isHoverBuffer"] = true
         })
```

Comment:
Given that this is the return result of `gemmiAtomsToCirclesSpheresInfo` and as far as I can see this function is only used here, would it make sense to update the function so that the items in `objects` already come out of it with these attributes defined? That way we don't need to loop through each item in `objects` and assign these values? I don't know how big this array of objects is but I guess this could save some time.

Reply:
I think the length of this is always 1.